docs/examples/random.md

```diff
@@ -0,0 +1,75 @@
+# opsdroid skill-random
+
+A skill for [opsdroid](https://github.com/opsdroid/opsdroid) to perform random events like flipping a coin or rolling a dice.
+## Creating skills
+
+Skills are designed to be simpler than other modules to ensure that it is easy to get started.
+
+To create a skill you need to at minimum create a single python file in your repository with the  `__init__.py`  name. For example the skill  `hello`  has a single file called  `__init__.py`.
```

Comment:
We don't care about generic information on 'creating skills', that already exists in the documentation, this is a tutorial on creating a specific skill. 

This section should be talking the user through how to create the right folder and files for the random chance skill they are creating.